code/API_definitions/application-profiles.yaml

```diff
@@ -385,6 +457,25 @@ components:
           $ref: "#/components/schemas/PacketErrorLossRate"
         jitter:
           $ref: "#/components/schemas/Jitter"
+      minProperties: 1
+
+    ComputeResourcesThresholds:
+      type: object
+      properties:
+        targetMinCPU:
+          $ref: "#/components/schemas/targetMinCPU"
+        targetMinMemory:
+          $ref: "#/components/schemas/targetMinMemory"
```

Comment:
Should these be min values or max values? If they are min values, that means we are allowing infinite over-provisioning of resources? Without a max value, we can't limit the amount of resources each application uses, and we can't calculate a total max value for multiple applications in case they run in a shared environment (multiple applications on a single Kubernetes cluster). From the viewpoint of managing resource allocation, it is better to require the max values that an application requires, rather than the min. In our platform, we treat any resource values as max values (resource limits in [Kubernetes speak](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)).